### DISCUSSION TITLE
Add "archive" button to mail client

### DISCUSSION THREAD

**Discussion by vampywiz17** [linuxdeepin/developer-center#3306] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hello there,

I use Gmail in Mail client. The mails ussually are move it to "archive" label, if i read/done it. Now your mail app, it is a bit complicated (right click > move > move to > chose "archived" label)

It would be good that a dedicated button on interface, or a right click menu entry (like outlook, and many other client) that we able to archive the mails only one click.

**Comment by BLumia** [comment]
@meiyixiang 需求翻译：当前邮件客户端归档邮件的步骤过于繁琐，用户希望能够通过更简单的方式来归档一个邮件，例如单独的界面按钮或层级没那么深的菜单来一键归档邮件。
